sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 5.3.2 (Unreleased)
 
+**New Features**
+
+- Added support for using `~azure.core.credentials.AzureSasCredential` as credential for authenticating producer and consumer clients.
```

Comment:
adding a note to say we have updated the azure-core dependency? as when we update uAMQP dependency we would add a note.
```suggestion
- Added support for using `~azure.core.credentials.AzureSasCredential` as credential for authenticating producer and consumer clients.

**Notes**
- Updated azure-core dependency to 1.13.0.
```